specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -0,0 +1,221 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetSetting": {
+            "$ref": "./examples/UpdateSetting-example.json"
+          }
+        }
+      }
+    },
+    "/settings": {
+      "get": {
+        "tags": [
+          "Settings"
+        ],
+        "operationId": "GetSettings",
+        "summary": "List account settings.",
+        "description": "Retrieves a list of all the available account settings that can be configured.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A response message containing a list of account settings with their associated value.",
+            "schema": {
+              "$ref": "#/definitions/SettingsListResult"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetSettings": {
+            "$ref": "./examples/ListSettings-example.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "UpdateSettingsRequest": {
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "The value of the pool setting."
+        }
+      },
+      "description": "The update settings request object.",
+      "required": [
+        "value"
+      ],
+      "type": "object"
+    },
+    "Setting": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The account setting to be updated"
+        },
+        "value": {
+          "type": "string",
+          "description": "The value of the pool setting."
+        },
+        "type": {
+          "enum": [
+            "boolean"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SettingTypeEnum",
+            "modelAsString": true
+          },
+          "description": "The type specifier of the value."
+        }
+      },
+      "required": [
+        "name",
+        "value"
+      ],
+      "type": "object"
+    },
+    "SettingsListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Setting"
+          },
+          "readOnly": true,
+          "description": "A response message containing a list of account settings with their associated value."
+        }
+      },
+      "description": "The settings list result.",
+      "type": "object"
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client API version.",
+      "x-ms-parameter-location": "client"
+    },
```

Comment:
Use the one in common.json once you've added the `x-ms-parameter-location` appropriately.